### DISCUSSION TITLE
Injection Triphasé

### DISCUSSION THREAD

**Discussion by mociulski** [FredM67/PVRouter-3-phase#74] [Q&A] [answered]
Bonjour Fred,

J'ai une installation electrique triphasé, avec production photovoltaïque branché sur une seule phase (L1). Je suis en train d'assembler un router mk2 triphasé avec deux sorties et je me demande comment le surplus est calculé par le router: individuel pour chaque phase? Ou en faisant l'addition des tous les phases ensemble?
Pour exemple, si sur L1 il y a 1kW production, L2 500w consomption et L3 200w consomption, le router va injecter 1kW ou 200w?

Merci,

**Comment by FredM67** [comment]
Salut,
Le routeur fonctionne tel que n'importe quel compteur français. Cf. [https://github.com/FredM67/PVRouter-3-phase/wiki/L'autoconsommation-en-triphas%C3%A9-pour-les-nuls](https://github.com/FredM67/PVRouter-3-phase/wiki/L'autoconsommation-en-triphas%C3%A9-pour-les-nuls)

Fred

**Comment by mociulski** [comment]
Merci,

J'aurais du RTFM :)

M.

**Reply by FredM67** [reply, marked as answer]
pour te rassurer, tu es loin, très loin, très très loin d'être le seul... il suffit de jeter un oeil sur le forum photovoltaique !

Ces questionnements reviennent interminablement, et chacun y va de sa théorie (toujours foireuse pour le coup), genre "oui, mais je n'ai pas de contrat de revente, alors comment le compteur peut compenser".. jusqu'à certains qui pensent que le Linky "tranfert" du courant d'une phase à l'autre !

Je pourrais presque en écrire un bouquin, lol
